README.md

```diff
@@ -489,12 +489,12 @@ under which circumstances:
 | HEAD |  |   |  Read |
 | GET |  |   |  Read |
 | POST | /foo/ | /foo/ exists | Append or Write on /foo/ + Write on resulting resource |
-| PUT | /foo/bar | /foo/ exists | Append or Write on /foo/ + Write on resulting resource |
-| PUT | /foo/bar | /foo/ gets created | Append or Write on / + Append or Write on /foo/ + Write on /foo/bar |
-| PATCH | /foo/bar | patch instructions contain deletions | Write on /foo/bar |
+| PUT | /foo/bar | /foo/ exists, will contain /foo/bar | Append or Write on /foo/ + Write on resulting resource |
+| PUT | /foo/bar | /foo/ gets created, will contain /foo/bar | Append or Write on / + Append or Write on /foo/ + Write on /foo/bar |
+| PATCH | /foo/bar | patch instructions contain deletions | Read and Write on /foo/bar |
 | PATCH | /foo/bar | patch instructions contain only insertions | Append or Write on /foo/bar |
 | DELETE | /foo/bar | | Write on /foo/bar |
-| DELETE | /foo/ | /foo/ is empty | Write on /foo/ |
+| DELETE | /foo/ | No resources other than /foo/ will get deleted | Write on /foo/ |
```

Comment:
How is this different than the `DELETE` note in the line above? Perhaps just adding the note about "No resource other than..." to the delete method above

Reply:
`DELETE /foo/bar` seems obviously different than `DELETE /foo/`... 

The first (tries to) `DELETE` a resource (`/foo/bar`).
The second (tries to) `DELETE` a container (`/foo/`).

That said, in several (but not all) other rows, the comment in the 3rd column is read with the titular preceding "if...", so "No resources other than /foo/ will get deleted" is definitely incorrect, and `/foo/ is empty` was closer to correct.  Possibly, `[if] /foo/ contains no other resources` would be better.  Possibly, the latter columns of this table should be reconsidered and rewritten overall, perhaps adding a column for "results" such that multiple `IF` conditions (e.g., `if /foo/ contains another resource`) could also be addressed in this table.

Reply:
All true! The root problem here is that the 'container is empty' thing implicitly ties WAC to Solid which we don't want. We should say 'For instance like when you're using WAC as part of Solid', and then link to https://github.com/solid/solid-spec/blob/master/api-rest.md#deleting-containers